cmd/ro/main.go

```diff
@@ -56,17 +66,25 @@ func registerFlags() {
 	flag.StringVar(&pswd, "password", "", "password")
 	flag.StringVar(&userEnv, "userenv", "RO_USER", "env variable for user name")
 	flag.StringVar(&pswdEnv, "pswdenv", "RO_PASS", "env variable for user password")
+	flag.StringVar(&pubKeyPath, "pubkey", "id_rsa.pub", "path to SSH public key")
 }
 
 func getUserCredentials() {
-	user = os.Getenv(userEnv)
-	pswd = os.Getenv(pswdEnv)
-	if user == "" || pswd == "" {
-		fmt.Print("Username:")
-		fmt.Scan(&user)
-		var err error
-		pswd, err = gopass.GetPass("Password:")
-		processError(err)
+	if user == "" {
+		user = os.Getenv(userEnv)
+		if user == "" {
+			fmt.Print("Username:")
+			fmt.Scan(&user)
```

Comment:
Scan blocks until non-whitespace is entered; the same behavior as before. Should this be changed?